DockerfileTest

```diff
@@ -10,6 +12,7 @@ WORKDIR /usr/src/app/
 RUN mvn install -Dmaven.test.skip -Djacoco.skip=true > /dev/null
 RUN cp ./rest-api/target/rest-api.jar /usr/src/run/
 RUN cp -r ./tools/docker/docker-test-artifacts/* /usr/src/run/
+RUN dos2unix /usr/src/run/qppConverterTest.sh
```

Comment:
Wait a second, that file already has LF line endings.  AKA, it already has "UNIX line endings".  Is it possible that the person on Windows is using some weird setting on `git` that forces CRLF line endings?

Reply:
Even so, maybe it's best to just keep this in here.  At worst it won't change the file.

Reply:
@halprin yes that' what is happening... but I figured we can keep this so the user has little to change when running our docker compose test.